### ISSUE TITLE
Android App v0.5.1 blank screen bug

### ISSUE THREAD

**Issue by martin0258** [ultralytics/hub#203]
### Search before asking

- [X] I have searched the HUB [issues](https://github.com/ultralytics/hub/issues) and found no similar bug report.

### HUB Component

Inference

### Bug

It's showing a blank screen even I've accepted all the permissions (including camera) required by th app.

Not sure if it's related to an old & close issue https://github.com/ultralytics/hub/issues/98
![image](https://user-images.githubusercontent.com/1118615/230600601-8796b6d7-3202-44de-b609-c6e0cf90332c.png)

### Environment

Device: Pixel 3a
Ultralytics Hub App version: 0.5.1

(Note: 0.4.1 was working)

**Comment by glenn-jocher** [comment]
@martin0258 hi! Thanks for the bug report! We will definitely look into this. 

@sergiossm is our lead mobile developer on this effort, and we have been updating the app for YOLOv8, so this may be related to this.

**Comment by glenn-jocher** [comment]
@martin0258 @sergiossm I'm able to reproduce the bug on a Xiaomi 13 with a new install of v0.5.1
https://user-images.githubusercontent.com/26833433/230793802-081547a8-5936-4451-a494-f5c36223707f.mov

**Comment by martin0258** [comment]
@glenn-jocher @sergiossm I'm a mobile app developer as well.

Below is the log messages I got on my Pixel 3a using `adb logcat`. Hope this helps:

```shell
# Get process id of the app
➜ adb shell ps | grep com.ultralytics.ultralytics_app | tr -s ' ' | cut -d' ' -f2
10609
# Get log messages of the app
➜ adb logcat | grep -F 10609                                                     
04-10 19:43:49.557  1059  1059 D Zygote  : Forked child process 10609
04-10 19:43:49.558  1870  2001 I ActivityManager: Start proc 10609:com.ultralytics.ultralytics_app/u0a489 for pre-top-activity {com.ultralytics.ultralytics_app/com.ultralytics.ultralytics_app.MainActivity}
04-10 19:43:49.577 10609 10609 I ultralytics_ap: Using CollectorTypeCC GC.
04-10 19:43:49.578 10609 10609 W ultralytics_ap: Mismatch between instruction set variant of device (ISA: Arm64 Feature string: -a53,crc,lse,fp16,dotprod,-sve) and features returned by the hardware (ISA: Arm64 Feature string: -a53,crc,lse,fp16,-dotprod,-sve)
04-10 19:43:49.587 10609 10609 E ultralytics_ap: Not starting debugger since process cannot load the jdwp agent.
04-10 19:43:49.599 10609 10609 D ProcessState: Binder ioctl to enable oneway spam detection failed: Invalid argument
04-10 19:43:49.631 10609 10609 D CompatibilityChangeReporter: Compat change id reported: 171979766; UID 10489; state: ENABLED
04-10 19:43:49.685 10609 10609 W ziparchive: Unable to open '/data/app/~~JeCF8Wr_FhaGgBm-BQ4Lhg==/com.ultralytics.ultralytics_app-0GyL0MQ6XHAo3x6yTTzcCA==/split_config.arm64_v8a.dm': No such file or directory
04-10 19:43:49.685 10609 10609 W ziparchive: Unable to open '/data/app/~~JeCF8Wr_FhaGgBm-BQ4Lhg==/com.ultralytics.ultralytics_app-0GyL0MQ6XHAo3x6yTTzcCA==/split_config.arm64_v8a.dm': No such file or directory
04-10 19:43:49.688 10609 10609 W ultralytics_ap: Entry not found
04-10 19:43:49.690 10609 10609 W ziparchive: Unable to open '/data/app/~~JeCF8Wr_FhaGgBm-BQ4Lhg==/com.ultralytics.ultralytics_app-0GyL0MQ6XHAo3x6yTTzcCA==/split_config.en.dm': No such file or directory
04-10 19:43:49.690 10609 10609 W ziparchive: Unable to open '/data/app/~~JeCF8Wr_FhaGgBm-BQ4Lhg==/com.ultralytics.ultralytics_app-0GyL0MQ6XHAo3x6yTTzcCA==/split_config.en.dm': No such file or directory
04-10 19:43:49.691 10609 10609 W ultralytics_ap: Entry not found
04-10 19:43:49.691 10609 10609 W ziparchive: Unable to open '/data/app/~~JeCF8Wr_FhaGgBm-BQ4Lhg==/com.ultralytics.ultralytics_app-0GyL0MQ6XHAo3x6yTTzcCA==/split_config.xxhdpi.dm': No such file or directory
04-10 19:43:49.692 10609 10609 W ziparchive: Unable to open '/data/app/~~JeCF8Wr_FhaGgBm-BQ4Lhg==/com.ultralytics.ultralytics_app-0GyL0MQ6XHAo3x6yTTzcCA==/split_config.xxhdpi.dm': No such file or directory
04-10 19:43:49.693 10609 10609 W ultralytics_ap: Entry not found
04-10 19:43:49.694 10609 10609 W ziparchive: Unable to open '/data/app/~~JeCF8Wr_FhaGgBm-BQ4Lhg==/com.ultralytics.ultralytics_app-0GyL0MQ6XHAo3x6yTTzcCA==/split_config.zh.dm': No such file or directory
04-10 19:43:49.694 10609 10609 W ziparchive: Unable to open '/data/app/~~JeCF8Wr_FhaGgBm-BQ4Lhg==/com.ultralytics.ultralytics_app-0GyL0MQ6XHAo3x6yTTzcCA==/split_config.zh.dm': No such file or directory
04-10 19:43:49.696 10609 10609 W ultralytics_ap: Entry not found
04-10 19:43:49.699 10609 10609 D nativeloader: Configuring classloader-namespace for other apk /data/app/~~JeCF8Wr_FhaGgBm-BQ4Lhg==/com.ultralytics.ultralytics_app-0GyL0MQ6XHAo3x6yTTzcCA==/base.apk:/data/app/~~JeCF8Wr_FhaGgBm-BQ4Lhg==/com.ultralytics.ultralytics_app-0GyL0MQ6XHAo3x6yTTzcCA==/split_config.arm64_v8a.apk:/data/app/~~JeCF8Wr_FhaGgBm-BQ4Lhg==/com.ultralytics.ultralytics_app-0GyL0MQ6XHAo3x6yTTzcCA==/split_config.en.apk:/data/app/~~JeCF8Wr_FhaGgBm-BQ4Lhg==/com.ultralytics.ultralytics_app-0GyL0MQ6XHAo3x6yTTzcCA==/split_config.xxhdpi.apk:/data/app/~~JeCF8Wr_FhaGgBm-BQ4Lhg==/com.ultralytics.ultralytics_app-0GyL0MQ6XHAo3x6yTTzcCA==/split_config.zh.apk. target_sdk_version=31, uses_libraries=, library_path=/data/app/~~JeCF8Wr_FhaGgBm-BQ4Lhg==/com.ultralytics.ultralytics_app-0GyL0MQ6XHAo3x6yTTzcCA==/lib/arm64:/data/app/~~JeCF8Wr_FhaGgBm-BQ4Lhg==/com.ultralytics.ultralytics_app-0GyL0MQ6XHAo3x6yTTzcCA==/base.apk!/lib/arm64-v8a:/data/app/~~JeCF8Wr_FhaGgBm-BQ4Lhg==/com.ultralytics.ultralytics_app-0GyL0MQ6XHAo3x6yTTzcCA==/split_config.arm64_v8a.apk!/lib/arm
04-10 19:43:49.705 10609 10609 W ultralytics_ap: JIT profile information will not be recorded: profile file does not exist.
04-10 19:43:49.705 10609 10609 W ultralytics_ap: JIT profile information will not be recorded: profile file does not exist.
04-10 19:43:49.705 10609 10609 W ultralytics_ap: JIT profile information will not be recorded: profile file does not exist.
04-10 19:43:49.705 10609 10609 W ultralytics_ap: JIT profile information will not be recorded: profile file does not exist.
04-10 19:43:49.716 10609 10609 V GraphicsEnvironment: ANGLE Developer option for 'com.ultralytics.ultralytics_app' set to: 'default'
04-10 19:43:49.719 10609 10609 V GraphicsEnvironment: ANGLE GameManagerService for com.ultralytics.ultralytics_app: false
04-10 19:43:49.720 10609 10609 V GraphicsEnvironment: Neither updatable production driver nor prerelease driver is supported.
04-10 19:43:49.722 10609 10609 D NetworkSecurityConfig: No Network Security Config specified, using platform default
04-10 19:43:49.722 10609 10609 D NetworkSecurityConfig: No Network Security Config specified, using platform default
04-10 19:43:49.732 10609 10609 I FirebaseApp: Device unlocked: initializing all Firebase APIs for app [DEFAULT]
04-10 19:43:49.749 10609 10609 I FirebaseCrashlytics: Initializing Firebase Crashlytics 18.2.11 for com.ultralytics.ultralytics_app
04-10 19:43:49.776 10609 10635 I DynamiteModule: Considering local module com.google.android.gms.measurement.dynamite:78 and remote module com.google.android.gms.measurement.dynamite:87
04-10 19:43:49.776 10609 10635 I DynamiteModule: Selected remote version of com.google.android.gms.measurement.dynamite, version >= 87
04-10 19:43:49.776 10609 10635 V DynamiteModule: Dynamite loader version >= 2, using loadModule2NoCrashUtils
04-10 19:43:49.785 10609 10635 W System  : ClassLoader referenced unknown path: 
04-10 19:43:49.785 10609 10635 D nativeloader: Configuring classloader-namespace for other apk . target_sdk_version=33, uses_libraries=, library_path=/data/app/~~hztlKuvEtSjwNUi5jhJcdw==/com.google.android.gms-rTGK27kUBxBrrk0qrSz51A==/lib/arm64:/data/app/~~hztlKuvEtSjwNUi5jhJcdw==/com.google.android.gms-rTGK27kUBxBrrk0qrSz51A==/base.apk!/lib/arm64-v8a, permitted_path=/data:/mnt/expand:/data/user/0/com.google.android.gms
04-10 19:43:49.788 10609 10609 I FirebaseInitProvider: FirebaseApp initialization successful
04-10 19:43:49.806 10609 10609 D WM-WrkMgrInitializer: Initializing WorkManager with default configuration.
04-10 19:43:49.819 10609 10635 W ultralytics_ap: ClassLoaderContext type mismatch. expected=PCL, found=DLC (PCL[] | DLC[];PCL[/data/app/~~JeCF8Wr_FhaGgBm-BQ4Lhg==/com.ultralytics.ultralytics_app-0GyL0MQ6XHAo3x6yTTzcCA==/base.apk*3475825224])
04-10 19:43:49.842 10609 10643 D CompatibilityChangeReporter: Compat change id reported: 160794467; UID 10489; state: ENABLED
04-10 19:43:49.905 10609 10650 I FA      : App measurement initialized, version: 75157
04-10 19:43:49.905 10609 10650 I FA      : To enable debug logging run: adb shell setprop log.tag.FA VERBOSE
04-10 19:43:49.905 10609 10650 I FA      : To enable faster debug mode event logging run:
04-10 19:43:49.905 10609 10650 I FA      :   adb shell setprop debug.firebase.analytics.app com.ultralytics.ultralytics_app
04-10 19:43:49.916 10609 10609 W ultralytics_app: type=1400 audit(0.0:14632): avc: denied { read } for name="max_map_count" dev="proc" ino=5065355 scontext=u:r:untrusted_app:s0:c233,c257,c512,c768 tcontext=u:object_r:proc_max_map_count:s0 tclass=file permissive=0 app=com.ultralytics.ultralytics_app
04-10 19:43:49.944 10609 10609 I AdrenoGLES-0: QUALCOMM build                   : 781e7d0, I46ff5fc46f
04-10 19:43:49.944 10609 10609 I AdrenoGLES-0: Build Date                       : 12/01/20
04-10 19:43:49.944 10609 10609 I AdrenoGLES-0: OpenGL ES Shader Compiler Version: EV031.31.04.01
04-10 19:43:49.944 10609 10609 I AdrenoGLES-0: Local Branch                     : QPR1
04-10 19:43:49.944 10609 10609 I AdrenoGLES-0: Remote Branch                    : 
04-10 19:43:49.944 10609 10609 I AdrenoGLES-0: Remote Branch                    : 
04-10 19:43:49.944 10609 10609 I AdrenoGLES-0: Reconstruct Branch               : 
04-10 19:43:49.944 10609 10609 I AdrenoGLES-0: Build Config                     : C P 11.0.1 AArch64
04-10 19:43:49.944 10609 10609 I AdrenoGLES-0: Driver Path                      : /vendor/lib64/egl/libGLESv2_adreno.so
04-10 19:43:49.946 10609 10609 I AdrenoGLES-0: PFP: 0x016ee190, ME: 0x00000000
04-10 19:43:49.949 10609 10609 W AdrenoUtils: <ReadGpuID_from_sysfs:197>: Failed to open /sys/class/kgsl/kgsl-3d0/gpu_model
04-10 19:43:49.950 10609 10609 W AdrenoUtils: <ReadGpuID:221>: Failed to read chip ID from gpu_model. Fallback to use the GSL path
04-10 19:43:49.950 10609 10609 D hw-ProcessState: Binder ioctl to enable oneway spam detection failed: Invalid argument
04-10 19:43:49.978 10609 10609 D DeviceInfoPlugin: Don't use TaskQueues.
04-10 19:43:50.027 10609 10650 I FA      : Tag Manager is not found and thus will not be used
04-10 19:43:50.200 10609 10609 W Choreographer: Frame time is 0.123987 ms in the future!  Check that graphics HAL is generating vsync timestamps using the correct timebase.
04-10 19:43:50.497 10609 10654 I Gralloc4: mapper 4.x is not supported
04-10 19:43:50.500 10609 10654 W Gralloc3: mapper 3.x is not supported
04-10 19:43:50.500  1870  1886 D ConnectivityService: requestNetwork for uid/pid:10489/10609 activeRequest: null callbackRequest: 9951 [NetworkRequest [ REQUEST id=9952, [ Capabilities: INTERNET&NOT_RESTRICTED&TRUSTED&NOT_VCN_MANAGED Uid: 10489 RequestorUid: 10489 RequestorPkg: com.ultralytics.ultralytics_app UnderlyingNetworks: Null] ]] callback flags: 0 order: 2147483647
04-10 19:43:50.513 10609 10654 W Gralloc4: allocator 4.x is not supported
04-10 19:43:50.516 10609 10654 W Gralloc3: allocator 3.x is not supported
04-10 19:43:50.551 10609 10609 W System.err: java.io.FileNotFoundException: coco.txt
04-10 19:43:50.551 10609 10609 W System.err: 	at android.content.res.AssetManager.nativeOpenAsset(Native Method)
04-10 19:43:50.551 10609 10609 W System.err: 	at android.content.res.AssetManager.open(AssetManager.java:881)
04-10 19:43:50.551 10609 10609 W System.err: 	at android.content.res.AssetManager.open(AssetManager.java:858)
04-10 19:43:50.551 10609 10609 W System.err: 	at p7.b.n(Unknown Source:17)
04-10 19:43:50.551 10609 10609 W System.err: 	at p7.b.<init>(Unknown Source:162)
04-10 19:43:50.551 10609 10609 W System.err: 	at p7.c.a(Unknown Source:10)
04-10 19:43:50.551 10609 10609 W System.err: 	at io.flutter.plugin.platform.n$a.f(Unknown Source:79)
04-10 19:43:50.551 10609 10609 W System.err: 	at g8.k$a.d(Unknown Source:232)
04-10 19:43:50.551 10609 10609 W System.err: 	at g8.k$a.c(Unknown Source:175)
04-10 19:43:50.551 10609 10609 W System.err: 	at h8.j$a.a(Unknown Source:17)
04-10 19:43:50.551 10609 10609 W System.err: 	at u7.c.l(Unknown Source:18)
04-10 19:43:50.551 10609 10609 W System.err: 	at u7.c.m(Unknown Source:20)
04-10 19:43:50.551 10609 10609 W System.err: 	at u7.c.i(Unknown Source:0)
04-10 19:43:50.551 10609 10609 W System.err: 	at u7.b.run(Unknown Source:12)
04-10 19:43:50.551 10609 10609 W System.err: 	at android.os.Handler.handleCallback(Handler.java:938)
04-10 19:43:50.551 10609 10609 W System.err: 	at android.os.Handler.dispatchMessage(Handler.java:99)
04-10 19:43:50.551 10609 10609 W System.err: 	at android.os.Looper.loopOnce(Looper.java:201)
04-10 19:43:50.551 10609 10609 W System.err: 	at android.os.Looper.loop(Looper.java:288)
04-10 19:43:50.551 10609 10609 W System.err: 	at android.app.ActivityThread.main(ActivityThread.java:7870)
04-10 19:43:50.551 10609 10609 W System.err: 	at java.lang.reflect.Method.invoke(Native Method)
04-10 19:43:50.551 10609 10609 W System.err: 	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:548)
04-10 19:43:50.551 10609 10609 W System.err: 	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:1003)
04-10 19:43:50.716 10609 10609 W PlatformViewsController: Unexpected platform view context for view ID 0; some functionality may not work correctly. When constructing a platform view in the factory, ensure that the view returned from PlatformViewFactory#create returns the provided context from getContext(). If you are unable to associate the view with that context, consider using Hybrid Composition instead.
```

**Comment by sergiossm** [comment]
Hey @martin0258, the issue is solved in version 0.5.2
